german/the-black-hack.md

```diff
@@ -113,7 +113,7 @@ Zeit im Spiel wird vor allem auf zwei Arten nachgehalten - **Momente** (Runden)
 
 ## Bewegung und Entfernung
 
-Anstatt genaue Werte nachzuhalten, benutzt TBH abstrakte Bereiche, mit denen die Entfernung dargestellt wird. **Benachbart**, **in der Nähe**, **abseits** und **entfernt**. Während eines Zugs können sich Charaktere irgendwo **in der Nähe** bewegen und ihre Handlung jederzeit während der Bewegung durchführen. Sie können ihre Handlung verfallen lassen, um sich **abseits** zu positionieren. Alles jenseits von **abseits** gilt als **entfernt** und würde drei Bewegungen erfordern.
+Anstatt genaue Werte nachzuhalten, benutzt TBH abstrakte Bereiche, mit denen die Entfernung dargestellt wird. **Dicht**, **Nah**, **Weit** und **Fern**. Während eines Zugs können sich Charaktere irgendwo **in der Nähe** bewegen und ihre Handlung jederzeit während der Bewegung durchführen. Sie können ihre Handlung verfallen lassen, um sich **abseits** zu positionieren. Alles jenseits von **abseits** gilt als **entfernt** und würde drei Bewegungen erfordern.
```

Comment:
Um es so zu halten wie im BH beabsichtigt, nämlich in Umgangssprachlich, bleibt eigentlich nur Option 2. Niemand redet bei Entfernungen von Schrittweite usw. Auch wenn Option 1. die schickere Lösung wäre, ist Nummer 2 eigentlich das was es eher sein soll. Daher muss ich für 2 stimmen.

Die Großschreibung ist doch nur in den Tabellen und Überschriften. Im Fließtext sind nur die Attributsabkürzungen immer groß. Daher weiß ich jetzt auf Anhieb nicht was du genau meinst.^^